### PR TITLE
OSDOCS-4255: clarify public and private API endpoints

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.11, 4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4255

Link to docs preview:
https://52119--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#rosa-sts-creating-cluster-customizations-ocm_rosa-sts-creating-a-cluster-with-customizations

QE review:
Defining public and private API endpoints. No new procedural information. No QE review required.

Additional information:

### CODE REVIEW

File: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc [hunk 1]
```diff
@@ -247,6 +247,15 @@ Alternatively, you can set your autoscaling preferences for the default machine
 +
 [IMPORTANT]
 ====
+The API endpoint cannot be changed between public and private after your cluster is created.
+====
++
+Public API endpoint:: Select *Public* if you do not want to restrict access to your cluster. You can access Kubernetes API endpoint and application routes from the internet.
```
Comment (arendej) [inline modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc:253]: > You can access Kubernetes API endpoint and application routes from the internet.

to

> You can access the Kubernetes API endpoint and application routes from the internet.

File: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc [hunk 2]
```diff
@@ -247,6 +247,15 @@ Alternatively, you can set your autoscaling preferences for the default machine
...
++
+Private API endpoint:: Select *Private* if you want to restrict access to to your cluster. You can access Kubernetes API endpoint and application routes from direct private connections only.
```
Comment (arendej) [inline modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc:255]: > if you want to restrict access to to your cluster. You can access Kubernetes API endpoint and application routes from direct private connections only.

to

> if you want to restrict network access to to your cluster. The Kubernetes API endpoint and application routes are accessible from direct private connections only.

**Review by kelbrown20 — COMMENTED** [review]
Looks great!
/lgtm

**Pull request merged**